server/src/models/map-library.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
 		const queryOptions = {
 			distinct: true,
 			where: {userID: userID},
-			limit: 20,
+			limit: 100,
```

Comment:
If we're updating map-library, map-favorite should probably also be updated, too.